doc/ug/getting_started_dv.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Getting Started with an OpenTitan Design Verification"
+---
+
+This document aims to enable a contributor to get started with a design verification (DV) effort within the OpenTitan project.
+While most of the focus is on development of a testbench from scratch, it should also be useful to understand how to contribute to an existing effort.
+Please refer to the [DV methodology]({{< relref "dv_methodology.md" >}}) document for information on how design verification is done in OpenTitan.
+
+## Stages of DV
```

Comment:
These can be dropped to first-level headings `#`.

Reply:
Prefer to match `getting_started_design.md`.